packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html

```diff
@@ -103,13 +103,35 @@
 
     <oui-tile-definition
         data-term="{{:: 'ovhManagerPccDashboardGeneralInformation_location_term' | translate }}"
-        data-description="{{:: $ctrl.bindings.location }}"
-    ></oui-tile-definition>
-
-    <oui-tile-definition
-        data-term="{{:: 'ovhManagerPccDashboardGeneralInformation_accessPolicy_term' | translate }}"
-        data-description="{{:: $ctrl.bindings.accessPolicy }}"
-    ></oui-tile-definition>
+    >
+        <oui-tile-description>
+            <dl class="mb-0">
+                <dt
+                    data-translate="ovhManagerPccDashboardGeneralInformation_location_zone"
+                ></dt>
+                <dd data-ng-bind=":: $ctrl.bindings.location"></dd>
+                <dt
+                    data-ng-if="!$ctrl.locationLoading && $ctrl.locationInfo"
```

Comment:
If you use `loading` prop you don't need `ngIf` checks anymore
```suggestion
                    data-ng-if="!$ctrl.locationLoading && $ctrl.locationInfo"
```